src/server.js

```diff
@@ -1,51 +1,16 @@
 import express from "express";
-import handlebars from "handlebars";
-import fs from "fs";
+import {engine} from "express-handlebars"
 import presentations from "./db/presentations.json" with { type: "json" };
 import searchPresentations from "./utils/search.js";
+import * as path from "node:path"
+import {fileURLToPath} from "node:url"
+const __dirname = path.dirname(fileURLToPath(import.meta.url))
 
 const app = express();
+app.engine(".hbs", engine({extname: ".hbs"}))
+app.set("view engine", ".hbs")
+app.set("views",path.resolve(__dirname,"./views"))
 
-const baseDir = "./src/";
```

Comment:
All of this is handled by the view engine, and is based on folder/file conventions